### PR TITLE
Unopresses Mentors

### PR DESCRIPTION
**Pull request by JamieD1**
Maybe unoppresses mentors and fixes dme edit

:cl:
tweak: Fixed mentors slots
/:cl:

### CODE REVIEW

**Comment by Reed0506** [comment]
based

**Comment by ynot01** [comment]
please attach https://github.com/yogstation13/Yogstation/issues/5169

**Review by alexkar598 — DISMISSED** [review]
Not the way to fix this, is donator already calls is mentor

https://github.com/yogstation13/Yogstation/blob/1c0e0ebe6784e51e79f72a36bdfb2b3c10aa6f37/yogstation/code/__HELPERS/mobs.dm#L36

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -129,7 +129,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			if(unlock_content)
 				max_save_slots = 8
 			// yogs start - Donor features
-			else if(is_donator(C) || (C.ckey in get_donators())) // the Latter handles race cases where the prefs are not fully loaded in, or GLOB.donators hasn't loaded in yet
+			else if(is_donator(C) || (C.ckey in get_donators()) || C.ckey in GLOB.mentor_datums)
```
Comment (Bibby0110) [inline code/modules/client/preferences.dm]:
```suggestion
			else if(is_donator(C) || (C.ckey in get_donators()) || (C.ckey in GLOB.mentor_datums)) // the Latter handles race cases where the prefs are not fully loaded in, or GLOB.donators hasn't loaded in yet, or GLOB.mentor_datums hasn't loaded in yet
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
Also do the thing I DM'ed you about you dingus

Also figure out which of your PRs you're doing this in dingus x2

**Review by fluffe9911 — APPROVED** [review]
mentors rise up

**Pull request merged**